app/helpers/application_helper.rb

```diff
@@ -13,4 +13,14 @@ def devise_mapping
     @devise_mapping ||= Devise.mappings[:user]
   end
 
+  def embedded_svg filename, options={}
```

Comment:
Use def with parentheses when there are parameters.<br>Surrounding space missing in default value assignment.